src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -1,9 +1,11 @@
 import {ActionTree, GetterTree, MutationTree} from 'vuex'
```

Comment:
@bzabos , I've request for you:
Could you in this particular line https://github.com/FLOIP/flow-builder/blob/VMO-4141-select-many-min-max-choice-editors/src/components/interaction-designer/block-types/MobilePrimitives_SelectOneResponseBlock.vue#L119 something like
```
    if (metadata.io_viamo.branchingType === UNIFIED) {
      this.handleBranchingTypeChangedToUnified({block: this.block})
    }
```
to make sure test values will be updated each time we have choices changes on UNIFIED branch type.

Reply:
Ah! You're brilliant. I saw this bug at the end of the night last night, but was too tired to fix it 😛

Reply:
Done!